hangman/test/hangman_test.exs

```diff
@@ -0,0 +1,86 @@
+defmodule HangmanTest do
```

Comment:
## +5

Yea! Tests!

It's interesting that you decided to test the implementation rather than the API. 

You might want to reconsider this approach, because it is coupling your test code to the internals of your game logic. If you decide to change the Game struct, you'd break all your tests, even though the API itself is unchanged. 

I personally do a mixture of both, but typically reserve internal tests such as these to those trickier bits of code which are hard to reach via the API but that I want to exercise thoroughly.